opencensus/trace/exporters/ocd_exporter.py

```diff
@@ -0,0 +1,217 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Export the trace spans to OpenCensusD"""
+
+import datetime
+import grpc
+
+from google.protobuf.timestamp_pb2 import Timestamp
+from google.protobuf.wrappers_pb2 import BoolValue, UInt32Value
+
+from opencensus.trace.exporters import base
+from opencensus.trace.exporters.transports import sync
+from opencensus.trace.exporters.gen.ocd.trace import trace_pb2
+from opencensus.trace.exporters.gen.ocd.exporter import exporter_pb2, exporter_pb2_grpc
+
+DEFAULT_ENDPOINT = 'localhost:50051'
+
+class OcdExporter(base.Exporter):
```

Comment:
I don't think we should use "OCD" as a name anywhere. It seems inappropriate since it's a psychiatric disorder any may cause offense or be hurtful.

Reply:
did you have any thoughts on how to name it? I think we should be consistent across languages.

Reply:
```proto3
service Export {
  rpc ExportSpan (stream ExportSpanRequest) returns (stream ExportSpanResponse) {}
  rpc ExportMetrics (stream ExportMetricsRequest) returns (stream ExportMetricsResponse) {}
}
```

the proto rpc is ExportSpan so SpanProtoExporter is an option (to avoid stuttering)